kyuubi-server/src/main/scala/org/apache/kyuubi/operation/KyuubiOperation.scala

```diff
@@ -36,6 +36,14 @@ import org.apache.kyuubi.util.ThriftUtils
 abstract class KyuubiOperation(opType: OperationType, session: Session)
   extends AbstractOperation(opType, session) {
 
+  private val operationClassName = getClass.getSimpleName
```

Comment:
use opType?

Reply:
OK, I will use `private val opTypeName = statement.toLowerCase` to replace it.

Reply:
I also find [LAUNCH_ENGINE](https://github.com/apache/incubator-kyuubi/blob/master/kyuubi-common/src/main/scala/org/apache/kyuubi/operation/OperationType.scala#L24-L26) maybe redundant. I see no anywhere else use it.

Reply:
The class name could be inconsistent during development, opType can be more traceable 
we can remove LAUNCH_ENGINE